### PR TITLE
0.9 rc1port, documentation fixes

### PR DESCRIPTION
**Pull request by atotic**

### CODE REVIEW

File: google-client-loader.html [hunk 1]
```diff
@@ -72,30 +70,28 @@
        */
 
       properties: {
+        x: String,
```
Comment (ebidel) [inline google-client-loader.html]: what are these?

Comment (atotic) [inline google-client-loader.html]: oops, part of my bug report for hydrolysis. Removed.

File: google-js-api.html [hunk 2]
```diff
@@ -10,7 +10,11 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../iron-jsonp-library/iron-jsonp-library.html">
 <!--
-Dynamically loads Google JavaScript API (gapi), firing the `js-api-load` event when ready.
+
```
Comment (ebidel) [inline google-js-api.html]: remove the blank comment

File: google-maps-api.html [hunk 3]
```diff
@@ -11,6 +11,10 @@
 <link rel="import" href="../iron-jsonp-library/iron-jsonp-library.html">
 
 <!--
```
Comment (ebidel) [inline google-maps-api.html]: remove

File: google-js-api.html (hunk 2 above)
Comment (atotic) [inline google-js-api.html]: done

File: google-maps-api.html (hunk 3 above)
Comment (atotic) [inline google-maps-api.html]: done

**Comment by ebidel** [comment]
Reviewed. LGTM

**Pull request merged**